docs/CONNECTION_RECOVERY.md

```
@@ -0,0 +1,283 @@
+## Usage Examples
+
+### Running with Automatic Fallback
+
+```bash
+# Runs normally if Neo4j is available
+# Falls back to mock mode if not
+poetry run python -m omega_kg.lifecycle --dry-run --no-email
+```
+
+**Output when Neo4j is unavailable:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)